packages/react-wrapper/package.json

```diff
@@ -42,7 +42,7 @@
   "peerDependencies": {
     "react": "^16.9.0",
     "react-dom": "^16.9.0",
-    "vue": "~2.6.0"
+    "vue": "^2.6.0"
```

Comment:
As the code does not change, we can still keep the 2.6 version as the oldest we support with react-wrapper